test/sql/json/test_json_copy.test_slow

```diff
@@ -99,12 +99,38 @@ copy conclusions from 'data/json/top_level_array.json'
 statement ok
 copy conclusions from 'data/json/top_level_array.json' (AUTO_DETECT TRUE)
 
+statement ok
+delete from conclusions;
+
+# and also if we say it's an array
+statement ok
+copy conclusions from 'data/json/top_level_array.json' (ARRAY TRUE)
```

Comment:
As mentioned above can we add a test with `ARRAY` without the `TRUE`?

Reply:
Good catch, this is now fixed and works as expected.